src/commands/info/about.ts

```diff
@@ -22,28 +22,28 @@ const execute = async (interaction: ChatInputCommandInteraction) => {
   }
 
   const aboutInfoEmbed = defaultEmbed(DefaultEmbedType.NORMAL)
-    .setTitle(`${t('title.title', { lng: 'en', ns: 'about_command' })}}`)
+    .setTitle(`${t('title', { lng: 'en', ns: 'about_command' })}}`)
     .addFields(
       {
-        name: t('fields.summary_title', {
+        name: t('field.summary_title', {
           //'Top-Tier Leveling Solution'
           lng: 'en',
           ns: 'about_command',
         }),
-        value: t('fields.summary_description', {
+        value: t('field.summary_description', {
           //'Elevate your Community to the next Level with Top-Tier Leveling, endless Customisability and more.'
           lng: 'en',
           ns: 'about_command',
         }),
         inline: false,
       },
       {
-        name: t('fields.server_title', {
+        name: t('field.server_title', {
           //'Join our Server'
           lng: 'en',
           ns: 'about_command',
         }),
-        value: t('fields.server_link', {
+        value: t('field.server_link', {
           //'[Join XP](https://discord.com/invite/ccTAnzw)'
```

Comment:
The links should be part of the translation! Only the "Join XP" should be translatable. Why would we translate a discord.com link?

Reply:
Idk but this is still not correct.

Only "Join XP" should be localized.

Reply:
??

Reply:
```ts
value: `[${t('field.server_link'), {...}}](https://discord.com/invite/ccTAnzw)`
```